core-bundle/src/DataContainer/DataContainerOperationsBuilder.php

```diff
@@ -39,7 +39,7 @@ public function __construct(
 
     public function __toString(): string
     {
-        if (null === $this->operations) {
+        if (null === $this->operations || [] === $this->operations) {
```

Comment:
```suggestion
        if (!$this->operations) {
```

This should cover both cases. But feel free to ignore.

Reply:
I did that, but the CS fixer or Rector changed it

Reply:
Actually, I used `empty()` and Rector changed that 🤔

Reply:
changed in 7ac258d3e2fdaa49e18f62d80f74fc5fb73c57af